### PR TITLE
fix: show warning custom dimension viewers

### PR DESCRIPTION
**Pull request by rephus**
Closes: https://github.com/lightdash/lightdash/issues/10676

thread: https://lightdash.slack.com/archives/C05UPNC0LGP/p1720704503869989?thread_ts=1720543597.835129&cid=C05UPNC0LGP

### Description:
Screenshot from 2024-07-11 16-36-12
Screenshot from 2024-07-11 16-36-34

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx [hunk 1]
```diff
@@ -775,22 +799,39 @@ const DashboardChartTileMain: FC<DashboardChartTileMainProps> = (props) => {
...
+                                                onClick={() =>
+                                                    handleCreateShareUrl(
+                                                        chartPathname,
+                                                        chartSearch,
+                                                    )
+                                                }
+                                            >
+                                                <Group>
+                                                    Explore from here
+                                                    <MantineIcon
```
Comment (joaoviana) [inline packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx:828]:
```suggestion
         {!cannotUseCustomDimensions &&<MantineIcon
```
so we don't display the alert if I can use custom dimensions

**Review by joaoviana — COMMENTED** [review]
One requested change.


And also tested with an interactive viewer:

image

File: packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx (hunk 1 above)
Comment (rephus) [reply packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx:828]: Thanks, I missed this

**Comment by rephus** [comment]
@joaoviana can I haz review ? :pray:

**Review by joaoviana — APPROVED** [review]

**Pull request merged**